### ISSUE TITLE
Website HTML code validation.

### ISSUE THREAD

**Issue by lockywolf** [cppalliance/temp-site#335]
Even though HTML is a "living standard" now, it still has validation tools (now provided by WHATWG, not w3c).

https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.boost.revsys.dev%2F&submit=Check

As of now it is listing 79 errors/warnings. Please, consider fixing them.

**Comment by vinniefalco** [comment]
Yeah well once the site is nearing completion of the first official public version we plan to validate and fix. And a round or two of page loading and SEO optimizations.
